bigflow/konfig.py

```diff
@@ -26,64 +27,58 @@ def current_env():
 class KonfigMeta(abc.ABCMeta):
 
     @staticmethod
-    def _copy_classvars_to_obj(klass, obj):
-        for k, v in klass.__dict__.items():
+    def prepare_dict(dct: tp.Dict):
+
+        anns = dct.setdefault('__annotations__', {})
+        for k, v in list(dct.items()):
+
             if k.startswith("_"):
-                # ignore private attributes
                 continue
-            elif hasattr(v, '__get__'):
-                # descriptor, pop out attribute from obj
-                obj.__dict__.pop(k, None)
-            else:
-                # normal value, put into obj
-                obj.__dict__[k] = v
-
-    def __new__(self, name, bases, dct):
+            elif isinstance(v, cached_property):
+                pass
+            elif hasattr(v, '__get__') or hasattr(v, '__set__') or hasattr(v, '__delete__'):
```

Comment:
Co to za sprawdzenie?

Reply:
Descriptor test https://docs.python.org/3/howto/descriptor.html#descriptor-protocol